doc/user/bgp.rst

```diff
@@ -2616,6 +2634,19 @@ the same behavior of using same next-hop and RMAC values.
 Enables or disables advertise-pip feature, specifiy system-IP and/or system-MAC
 parameters.
 
+EVPN advertise-svi
+^^^^^^^^^^^^^^^^
```

Comment:
nit: Two more `^^` to adjust with the title :)

Reply:
Good catch, thanks @ton31337 . Fixed!